test/specs/scenarios/document.js

```diff
@@ -189,14 +189,71 @@ export const documentScenarios = async (lang) => {
           confirm,
           'identity_card_with_glare.jpg'
         )
-        confirm.verifyGlareDetectedWarning(copy)
+        confirm.verifyImageQualityWarning(copy, 'glare')
+
+        // 1st retake
+        confirm.clickRedoButton()
+        uploadFileAndClickConfirmButton(
+          documentUpload,
+          confirm,
+          'identity_card_with_cut-off.png'
+        )
+        confirm.verifyImageQualityWarning(copy, 'cut-off')
+
+        // 2nd retake
+        confirm.clickRedoButton()
+        uploadFileAndClickConfirmButton(
+          documentUpload,
+          confirm,
+          'identity_card_with_glare.jpg'
+        )
+        confirm.verifyImageQualityWarning(copy, 'glare')
+
+        // Process all the way
+        confirm.confirmBtn().isDisplayed()
         confirm.clickConfirmButton()
+      })
+
+      it('should return image quality message on back id card', async () => {
+        driver.get(`${baseUrl}&async=false&useUploader=true`)
```

Comment:
`useUploader` isn't necessary here or in line 181 actually. It's only intended for when a UI test needs to upload a Selfie photo instead of the camera UI.